common/check_messages_to_submit

```diff
@@ -32,31 +31,24 @@ else:
 
 queue_sql = """SELECT COUNT(*) FROM {schema}messages""".format(schema=schema)
 
-PROM_SERVERS = config_get('monitor', 'prometheus_servers', raise_exception=False, default='')
-if PROM_SERVERS != '':
-    PROM_SERVERS = PROM_SERVERS.split(',')
-
 if __name__ == "__main__":
     try:
-        registry = CollectorRegistry()
         session = get_session()
-        result = session.execute(queue_sql).fetchall()
-        print('queues.messages %s' % result[0][0])
-        probe_metrics.gauge(name='queues.messages').set(result[0][0])
-        Gauge('hermes_queues_messages', '', registry=registry).set(result[0][0])
-
-        if len(PROM_SERVERS):
-            for server in PROM_SERVERS:
-                try:
-                    push_to_gateway(server.strip(), job='check_messages_to_submit', registry=registry)
-                except:
-                    continue
-
-        if result[0][0] > 100000:
+        with PrometheusPusher() as manager:
+            result = session.execute(sql_text(queue_sql)).fetchall()
+            message_count = result[0][0]
+            print(f"Messages in queue: {message_count}")
+
+            manager.gauge(
+                "messages_to_submit.queues.messages", 
```

Comment:
Again this changes the metric value. The job label in prometheus will say where this comes from, but it still might be a good idea.